tests_e2e/test_app.py

```diff
@@ -28,11 +31,15 @@ def test_app():
 
 @pytest.fixture(scope='module')
 def driver():
-    with webdriver.Firefox() as driver:
+    opts = webdriver.ChromeOptions()
+    opts.add_argument('--headless')
+    opts.add_argument('--no-sandbox')
+    opts.add_argument('--disable-dev-shm-usage')
+    with webdriver.Chrome('./chromedriver', options=opts) as driver:
```

Comment:
You can write `webdriver.Chrome(options=opts)` so that it **also** checks the PATH rather than requiring it in the current directory.